tools/ci/regen_certs.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import base64
+import logging
+import subprocess
+
+
+logging.basicConfig()
```

Comment:
This is a bad practice, as it'd cause side effects when importing this file (it's quite a common mistake; even Google AppEngine SDK once had this which broke lots of people's logging setup). Move it to the run-time entrypoint.